packages/test-cases/cases/cli/wasm/build-cmd/assemblyscript/015-resource-files/polywrap.yaml

```diff
@@ -0,0 +1,11 @@
+format: 0.3.0
+project:
+  name: test-project
+  type: wasm/assemblyscript
+source:
+  module: ./src/index.ts
+  schema: ./src/schema.graphql
+resources:
```

Comment:
I'd propose having a single directory for resources:
```
resources: ./resources
```

```
resources: ./files
```
Instead of having list of files and directories.
This allows the developer to add files to the build artifacts despite them having the same name as the files in the project (e.g. having separate project and build README's that are both named README.md)

Reply:
Okay I agree with this approach, nice 👍

Reply:
Okay this has been implemented, super nice simplification!